vulkan.symbols.api

```diff
@@ -0,0 +1,207 @@
+vkAcquireNextImage2KHR
```

Comment:
Long term, we should try to get something to ensure that this matches vulkan-1.def. It's not that important right now, but whenever the next Vulkan minor release comes out it would be a good idea to have that.

Reply:
Sure. I'll file an issue after we merge this.